tests/shell/shell_recorder/globalplugins.esr

```diff
@@ -1,32 +0,0 @@
-Mountpoint: system/
-DiffType: File
-
-
-< set system/elektra/globalplugins ""
```

Comment:
I just disabled these tests, since they currently fail and I do not know if the current output is correct. The text below shows the current output of the tests.

- `testshell_globalplugins`:

  ```
  104: Test command: Elektra/build/tests/shell/shell_recorder/shell_recorder.sh "Elektra/tests/shell/shell_recorder/globalplugins.esr"
  104: Test timeout computed to be: 1500
  104: protocol file: /var/folders/hx/flbncdhj4fs87095gzxvnj3h0000gn/T/elektraenv.XXXXXXXXX.v9Y9Mh8K
  104: kdb set system/elektra/globalplugins ""
  104: kdb set system/elektra/globalplugins/postcommit ""
  104: kdb set system/elektra/globalplugins/postcommit/init list
  104: kdb set system/elektra/globalplugins/postcommit/init/user list
  104: kdb set system/elektra/globalplugins/postcommit/init/user/placements ""
  104: kdb set system/elektra/globalplugins/postcommit/init/user/placements/set "presetstorage presetcleanup precommit postcommit"
  104: kdb set system/elektra/globalplugins/postcommit/init/user/placements/get "pregetstorage postgetstorage postgetcleanup"
  104: kdb set system/elektra/globalplugins/postcommit/init/user/placements/error "prerollback postrollback"
  104: kdb set system/elektra/globalplugins/postcommit/init/user/plugins ""
  104: kdb set system/elektra/globalplugins/postcommit/init/user/plugins/#0 tracer
  104: kdb set system/elektra/globalplugins/postcommit/init/user/plugins/#0/placements tracer
  104: kdb set system/elektra/globalplugins/postcommit/init/user/plugins/#0/placements/set "presetstorage precommit postcommit"
  104: kdb set system/elektra/globalplugins/postcommit/init/user/plugins/#1 timeofday
  104: kdb set system/elektra/globalplugins/postcommit/init/user/plugins/#1/placements timeofday
  104: kdb set system/elektra/globalplugins/postcommit/init/user/plugins/#1/placements/set "presetstorage precommit postcommit"
  104: kdb set system/elektra/globalplugins/postrollback list
  104: kdb set system/elektra/globalplugins/precommit list
  104: kdb set system/elektra/globalplugins/pregetstorage list
  104: kdb set system/elektra/globalplugins/postgetcleanup list
  104: kdb set system/elektra/globalplugins/postgetstorage list
  104: kdb set system/elektra/globalplugins/presetstorage list
  104: kdb set system/elektra/globalplugins/presetcleanup list
  104: kdb set system/elektra/globalplugins/prerollback list
  104: kdb set system/globaltestkey
  104:
  104: ERROR - STDERR:
  104: open	0000000000	di	0000000000
  104: set	0000000027	di	0000000027	pos	presetstorage
  104: set	0000000509	di	0000000482	pos	precommit
  104: close	0000000562	di	0000000053
  104: doesn't match (set.*di.*pos.*){3}
  104:
  104: tracer: open(0x7fc824407d40, system = /etc/kdb/default.ecf): 0
  104: tracer: set(0x7fc824407d40, system, /etc/kdb/default.ecf): 0
  104: tracer: close(0x7fc824407d40,  = )
  104: tracer: open(0x7fe4be41c2d0, system = /dev/stdin): 0
  104: tracer: set(0x7fe4be41c2d0, system, /dev/stdin): 0
  104: tracer: set(0x7fe4be41c2d0, system, /etc/kdb/default.ecf): 0
  104: tracer: close(0x7fe4be41c2d0,  = )
  104: shell_recorder Elektra/tests/shell/shell_recorder/globalplugins.esr RESULTS: 3 test(s) done
  104:  1 error(s).
  1/1 Test #104: testshell_globalplugins ..........***Failed    1.41 sec

  0% tests passed, 1 tests failed out of 1

  Label Time Summary:
  kdbtests    =   1.41 sec (1 test)
  memleak     =   1.41 sec (1 test)

  Total Test time (real) =   1.42 sec

  The following tests FAILED:
  	104 - testshell_globalplugins (Failed)
  Errors while running CTest
  ```

- `testshell_globaltest`

  ```
  105: Test command: Elektra/build/tests/shell/shell_recorder/shell_recorder.sh "Elektra/tests/shell/shell_recorder/globaltest.esr"
  105: Test timeout computed to be: 1500
  105: protocol file: /var/folders/hx/flbncdhj4fs87095gzxvnj3h0000gn/T/elektraenv.XXXXXXXXX.UYw2oYMR
  105: kdb set system/elektra/globalplugins ""
  105: kdb set system/elektra/globalplugins/postcommit list
  105: kdb set system/elektra/globalplugins/postcommit/user list
  105: kdb set system/elektra/globalplugins/postcommit/user/placements ""
  105: kdb set system/elektra/globalplugins/postcommit/user/placements/set "presetstorage presetcleanup precommit postcommit"
  105: kdb set system/elektra/globalplugins/postcommit/user/placements/get "pregetstorage postgetstorage postgetcleanup"
  105: kdb set system/elektra/globalplugins/postcommit/user/placements/error "prerollback postrollback"
  105: kdb set system/elektra/globalplugins/postcommit/user/plugins ""
  105: kdb set system/elektra/globalplugins/postcommit/user/plugins/#0 tracer
  105: kdb set system/elektra/globalplugins/postcommit/user/plugins/#0/placements tracer
  105: kdb set system/elektra/globalplugins/postcommit/user/plugins/#0/placements/set "presetstorage precommit postcommit"
  105: kdb set system/elektra/globalplugins/postcommit/user/plugins/#1 timeofday
  105: kdb set system/elektra/globalplugins/postcommit/user/plugins/#1/placements timeofday
  105: kdb set system/elektra/globalplugins/postcommit/user/plugins/#1/placements/set "presetstorage precommit postcommit"
  105: kdb set system/elektra/globalplugins/postrollback list
  105: kdb set system/elektra/globalplugins/precommit list
  105: kdb set system/elektra/globalplugins/pregetstorage list
  105: kdb set system/elektra/globalplugins/postgetcleanup list
  105: kdb set system/elektra/globalplugins/postgetstorage list
  105: kdb set system/elektra/globalplugins/presetstorage list
  105: kdb set system/elektra/globalplugins/presetcleanup list
  105: kdb set system/elektra/globalplugins/prerollback list
  105: kdb set system/globaltestkey
  105:
  105: ERROR - STDERR:
  105: open	0000000000	di	0000000000
  105: get	0000000027	di	0000000027	pos	pregetstorage
  105: get	0000000150	di	0000000123	pos	postgetstorage
  105: close	0000000253	di	0000000103
  105: open	0000000001	di	0000000001
  105: get	0000000012	di	0000000011	pos	pregetstorage
  105: set	0000000049	di	0000000037	pos	presetstorage
  105: set	0000000425	di	0000000376	pos	precommit
  105: close	0000000478	di	0000000053
  105: doesn't match (set.*di.*pos.*){3}
  105:
  105: tracer: open(0x7fd46ae0bce0, system = /etc/kdb/default.ecf): 0
  105: tracer: get(0x7fd46ae0bce0, system, /etc/kdb/default.ecf): user/overrides/test user/sw/elektra/examples/kdb-complete/% user/sw/elektra/examples/kdb-complete/#array_1 user/sw/elektra/examples/kdb-complete/buzz user/sw/elektra/examples/kdb-complete/level1 user/sw/elektra/examples/kdb-complete/lvl1/lvl2 user/sw/elektra/examples/kdb-complete/lvl1/lvl2/lvl3/lvl4/lvl5 user/sw/kdb/current/plugins user/sw/org/myapp/#0/current user/sw/org/myapp/#0/profile user/sw/org/myapp/#0/testprofile user/sw/org/myapp/#0/testprofile/testkey 12
  105: tracer: get(0x7fd46ae0bce0, /sw/elektra/kdb/#0/current, ): user/sw/org/myapp/#0/current user/sw/org/myapp/#0/profile user/sw/org/myapp/#0/testprofile user/sw/org/myapp/#0/testprofile/testkey 4
  105: tracer: close(0x7fd46ae0bce0,  = )
  105: tracer: open(0x7fd46ae0ad80, system/elektra/modules/dump = ): 0
  105: tracer: get(0x7fd46ae0ad80, system/elektra/modules/dump, ): system/elektra/modules/list system/elektra/modules/list/exports system/elektra/modules/list/exports/addPlugin system/elektra/modules/list/exports/close system/elektra/modules/list/exports/editPlugin system/elektra/modules/list/exports/error system/elektra/modules/list/exports/get system/elektra/modules/list/exports/open system/elektra/modules/list/exports/set system/elektra/modules/list/infos system/elektra/modules/list/infos/author system/elektra/modules/list/infos/description system/elektra/modules/list/infos/licence system/elektra/modules/list/infos/needs system/elektra/modules/list/infos/placements system/elektra/modules/list/infos/provides system/elektra/modules/list/infos/status system/elektra/modules/list/infos/version system/elektra/modules/resolver_fm_hpu_b system/elektra/modules/resolver_fm_hpu_b/constants system/elektra/modules/resolver_fm_hpu_b/constants/ELEKTRA_VARIANT_BASE system/elektra/modules/resolver_fm_hpu_b/constants/ELEKTRA_VARIANT_SYSTEM system/elektra/modules/resolver_fm_hpu_b/constants/ELEKTRA_VARIANT_USER system/elektra/modules/resolver_fm_hpu_b/constants/KDB_DB_DIR system/elektra/modules/resolver_fm_hpu_b/constants/KDB_DB_HOME system/elektra/modules/resolver_fm_hpu_b/constants/KDB_DB_SPEC system/elektra/modules/resolver_fm_hpu_b/constants/KDB_DB_SYSTEM system/elektra/modules/resolver_fm_hpu_b/constants/KDB_DB_USER system/elektra/modules/resolver_fm_hpu_b/exports system/elektra/modules/resolver_fm_hpu_b/exports/checkfile system/elektra/modules/resolver_fm_hpu_b/exports/close system/elektra/modules/resolver_fm_hpu_b/exports/error system/elektra/modules/resolver_fm_hpu_b/exports/get system/elektra/modules/resolver_fm_hpu_b/exports/open system/elektra/modules/resolver_fm_hpu_b/exports/set system/elektra/modules/resolver_fm_hpu_b/infos system/elektra/modules/resolver_fm_hpu_b/infos/author system/elektra/modules/resolver_fm_hpu_b/infos/description system/elektra/modules/resolver_fm_hpu_b/infos/licence system/elektra/modules/resolver_fm_hpu_b/infos/needs system/elektra/modules/resolver_fm_hpu_b/infos/placements system/elektra/modules/resolver_fm_hpu_b/infos/provides system/elektra/modules/resolver_fm_hpu_b/infos/status system/elektra/modules/resolver_fm_hpu_b/infos/version system/elektra/version system/elektra/version/constants system/elektra/version/constants/KDB_VERSION system/elektra/version/constants/KDB_VERSION_MAJOR system/elektra/version/constants/KDB_VERSION_MICRO system/elektra/version/constants/KDB_VERSION_MINOR system/elektra/version/constants/SO_VERSION system/elektra/version/infos system/elektra/version/infos/author system/elektra/version/infos/description system/elektra/version/infos/licence system/elektra/version/infos/version system/globaltestkey system/overrides/test 58
  105: tracer: set(0x7fd46ae0ad80, system, /etc/kdb/default.ecf): 0
  105: tracer: close(0x7fd46ae0ad80,  = )
  105: tracer: open(0x7f858360add0, system = /etc/kdb/default.ecf): 0
  105: tracer: get(0x7f858360add0, system, /etc/kdb/default.ecf): user/overrides/test user/sw/elektra/examples/kdb-complete/% user/sw/elektra/examples/kdb-complete/#array_1 user/sw/elektra/examples/kdb-complete/buzz user/sw/elektra/examples/kdb-complete/level1 user/sw/elektra/examples/kdb-complete/lvl1/lvl2 user/sw/elektra/examples/kdb-complete/lvl1/lvl2/lvl3/lvl4/lvl5 user/sw/kdb/current/plugins user/sw/org/myapp/#0/current user/sw/org/myapp/#0/profile user/sw/org/myapp/#0/testprofile user/sw/org/myapp/#0/testprofile/testkey 12
  105: tracer: get(0x7f858360add0, /sw/elektra/kdb/#0/current, ): user/sw/org/myapp/#0/current user/sw/org/myapp/#0/profile user/sw/org/myapp/#0/testprofile user/sw/org/myapp/#0/testprofile/testkey 4
  105: tracer: close(0x7f858360add0,  = )
  105: tracer: open(0x7f8583609d20, system/elektra/modules/dump = ): 0
  105: tracer: get(0x7f8583609d20, system/elektra/modules/dump, ): system/elektra/modules/list system/elektra/modules/list/exports system/elektra/modules/list/exports/addPlugin system/elektra/modules/list/exports/close system/elektra/modules/list/exports/editPlugin system/elektra/modules/list/exports/error system/elektra/modules/list/exports/get system/elektra/modules/list/exports/open system/elektra/modules/list/exports/set system/elektra/modules/list/infos system/elektra/modules/list/infos/author system/elektra/modules/list/infos/description system/elektra/modules/list/infos/licence system/elektra/modules/list/infos/needs system/elektra/modules/list/infos/placements system/elektra/modules/list/infos/provides system/elektra/modules/list/infos/status system/elektra/modules/list/infos/version system/elektra/modules/resolver_fm_hpu_b system/elektra/modules/resolver_fm_hpu_b/constants system/elektra/modules/resolver_fm_hpu_b/constants/ELEKTRA_VARIANT_BASE system/elektra/modules/resolver_fm_hpu_b/constants/ELEKTRA_VARIANT_SYSTEM system/elektra/modules/resolver_fm_hpu_b/constants/ELEKTRA_VARIANT_USER system/elektra/modules/resolver_fm_hpu_b/constants/KDB_DB_DIR system/elektra/modules/resolver_fm_hpu_b/constants/KDB_DB_HOME system/elektra/modules/resolver_fm_hpu_b/constants/KDB_DB_SPEC system/elektra/modules/resolver_fm_hpu_b/constants/KDB_DB_SYSTEM system/elektra/modules/resolver_fm_hpu_b/constants/KDB_DB_USER system/elektra/modules/resolver_fm_hpu_b/exports system/elektra/modules/resolver_fm_hpu_b/exports/checkfile system/elektra/modules/resolver_fm_hpu_b/exports/close system/elektra/modules/resolver_fm_hpu_b/exports/error system/elektra/modules/resolver_fm_hpu_b/exports/get system/elektra/modules/resolver_fm_hpu_b/exports/open system/elektra/modules/resolver_fm_hpu_b/exports/set system/elektra/modules/resolver_fm_hpu_b/infos system/elektra/modules/resolver_fm_hpu_b/infos/author system/elektra/modules/resolver_fm_hpu_b/infos/description system/elektra/modules/resolver_fm_hpu_b/infos/licence system/elektra/modules/resolver_fm_hpu_b/infos/needs system/elektra/modules/resolver_fm_hpu_b/infos/placements system/elektra/modules/resolver_fm_hpu_b/infos/provides system/elektra/modules/resolver_fm_hpu_b/infos/status system/elektra/modules/resolver_fm_hpu_b/infos/version system/elektra/version system/elektra/version/constants system/elektra/version/constants/KDB_VERSION system/elektra/version/constants/KDB_VERSION_MAJOR system/elektra/version/constants/KDB_VERSION_MICRO system/elektra/version/constants/KDB_VERSION_MINOR system/elektra/version/constants/SO_VERSION system/elektra/version/infos system/elektra/version/infos/author system/elektra/version/infos/description system/elektra/version/infos/licence system/elektra/version/infos/version system/globaltestkey system/overrides/test 58
  105: tracer: set(0x7f8583609d20, system, /dev/stdin): 0
  105: tracer: set(0x7f8583609d20, system, /etc/kdb/default.ecf): 0
  105: tracer: close(0x7f8583609d20,  = )
  105: shell_recorder Elektra/tests/shell/shell_recorder/globaltest.esr RESULTS: 3 test(s) done
  105:  1 error(s).
  1/1 Test #105: testshell_globaltest .............***Failed    1.38 sec

  0% tests passed, 1 tests failed out of 1

  Label Time Summary:
  kdbtests    =   1.38 sec (1 test)
  memleak     =   1.38 sec (1 test)

  Total Test time (real) =   1.39 sec

  The following tests FAILED:
  	105 - testshell_globaltest (Failed)
  Errors while running CTest
  ```